doc/snippets/Microsoft.Data.SqlClient/SqlCommand.xml

```diff
@@ -1,3108 +1,4330 @@
+            </listheader>
+            <item>
+              <term><see cref="M:Microsoft.Data.SqlClient.SqlCommand.BeginExecuteNonQuery" /></term>
+              <description>
+                Initiates the asynchronous execution of the Transact-SQL statement or stored procedure that is described by this <see cref="T:Microsoft.Data.SqlClient.SqlCommand" />, generally executing commands such as INSERT, DELETE, UPDATE, and SET statements. Each call to <see cref="M:Microsoft.Data.SqlClient.SqlCommand.BeginExecuteNonQuery" /> must be paired with a call to <see cref="M:Microsoft.Data.SqlClient.SqlCommand.EndExecuteNonQuery" /> which finishes the operation, typically on a separate thread.
+              </description>
+            </item>
+            <item>
+              <term><see cref="M:Microsoft.Data.SqlClient.SqlCommand.BeginExecuteReader" /></term>
+              <description>
+                Initiates the asynchronous execution of the Transact-SQL statement or stored procedure that is described by this <see cref="T:Microsoft.Data.SqlClient.SqlCommand" /> and retrieves one or more results sets from the server. Each call to <see cref="M:Microsoft.Data.SqlClient.SqlCommand.BeginExecuteReader" /> must be paired with a call to <see cref="M:Microsoft.Data.SqlClient.SqlCommand.EndExecuteReader" /> which finishes the operation, typically on a separate thread.
+              </description>
+            </item>
+            <item>
+              <term><see cref="M:Microsoft.Data.SqlClient.SqlCommand.BeginExecuteXmlReader" /></term>
+              <description>
+                Initiates the asynchronous execution of the Transact-SQL statement or stored procedure that is described by this <see cref="T:Microsoft.Data.SqlClient.SqlCommand" />. Each call to <see cref="M:Microsoft.Data.SqlClient.SqlCommand.BeginExecuteXmlReader" /> must be paired with a call to <see cref="M:Microsoft.Data.SqlClient.SqlCommand.EndExecuteXmlReader" />, which finishes the operation, typically on a separate thread, and returns an <see cref="T:System.Xml.XmlReader" /> object.
+              </description>
+            </item>
+            <item>
+              <term><see cref="M:Microsoft.Data.SqlClient.SqlCommand.ExecuteReader" /></term>
+              <description>
+                Executes commands that return rows. For increased performance, <see cref="M:Microsoft.Data.SqlClient.SqlCommand.ExecuteReader" /> invokes commands using the Transact-SQL <c>sp_executesql</c> system stored procedure. Therefore, <see cref="M:Microsoft.Data.SqlClient.SqlCommand.ExecuteReader" /> might not have the effect that you want if used to execute commands such as Transact-SQL SET statements.
+              </description>
+            </item>
+            <item>
+              <term><see cref="M:Microsoft.Data.SqlClient.SqlCommand.ExecuteNonQuery" /></term>
+              <description>Executes commands such as Transact-SQL INSERT, DELETE, UPDATE, and SET statements.</description>
```

Comment:
I think the SQL keywords should be quoted.

Reply:
Ehhhh.... there's a lot of these and I don't know if it's really worth it.

Reply:
Though for the record, it wouldn't be a cref, b/c those are used for links to other documentation in the same language. It would have to be wrapped with `<c></c>`